careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Job Application Process</h2>
+                <p class="short-desc">
+                    The application can be tricky.
+                    We can help.
+                    Learn how the application process works.
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn about job application process
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Working at the CFPB</h2>
+                <p class="short-desc">
+                    Help us continue to grow
+                    and develop our 21st century agency.
+                    The work you do will directly impact the lives of consumers
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn what it's like to work at the CFPB
+                    </a>
+                </p>
+            </section>
+        </div>
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Follow us on LinkedIn</h2>
+                <p class="short-desc">
+                    The CFPB is one of the most searched for agencies
+                    in the federal government.
+                    Connect with us to stay updated on the work
+                    we do and new opportunities to be a part of it.
+                </p>
+                <p class="short-desc">
+                    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+                       class="jump-link jump-link__right">
+                        Go to our LinkedIn
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Provide Us With Feedback</h2>
+                <p class="short-desc">
+                    {# TODO: Have Content look at this paragraph
+                    and the weird link break. #}
+
+                    We take pride in our feedback-centered culture
+                    that helps us recruit great people.
+                    If you have any feedback, or any specific questions,
+                    let us know at:
+                </p>
+                <p class="short-desc">
+                    <ul class="list list__icons">
+                        <li class="list_item">
+                            <span class="cf-icon cf-icon-email list_icon"></span>
+                            <span class="h5 list_text">Email:</span>
```

Comment:
Confirm whether `Email:` needs colon. I know /contact-us/ has it, but the mocks don't have it. cc @duelj